index.html

```diff
@@ -14,24 +14,78 @@
   mkd: null,
   xml: null,
   txt: null,
-  name: null
+  name: null,
+  timer: null
+}
+
+// Save the doc.  If 'cur' is specified, this is a new doc, and we need
+// to store its identifier.
```

Comment:
A few more notes on the storage model here would be appreciated.  When are things stored / loaded?  What is the intended semantic of `CURRENT`?